### DISCUSSION TITLE
pm2貌似无法正常启动

### DISCUSSION THREAD

**Discussion by StarryFX** [Tsuk1ko/CQ-picfinder-robot#146] [问题求助]
文件未动过，某次开机后启动机器人发现无法正常启动，对比了我朋友的正常启动，认为是pm2的问题
![QQ图片20210211222058](https://user-images.githubusercontent.com/78916254/107649132-1cecdf00-6cb8-11eb-8d05-55a26aca946a.png)
寻找了求助区的类似的案例还未解决，如有打扰，实属抱歉（如有解决方法，请尽量详细的叙述过程，本人新手，基本不懂，再次感谢能回答我的大佬）

**Comment by Tsuk1ko** [comment]
直接启动可以而用pm2就不行么

**Comment by StarryFX** [comment]
不是很懂直接启动...就是按照介绍步骤这样启动的，没改过任何东西，试过npx pm2 kill之后再yarn，会显示正常启动pm2，但机器人没有上线，重启后又变成这样了...

**Comment by Tsuk1ko** [comment]
@StarryFX npm run test

**Comment by StarryFX** [comment]
![image](https://user-images.githubusercontent.com/78916254/107919071-b8769c00-6fa5-11eb-83c5-08d42aaa411a.png)
写了这个之后紧接着启动就显示正常，但机器人依旧没有回应

**Comment by Tsuk1ko** [comment]
@StarryFX 这个有错就是不能正常启动了

看起来是依赖问题，把 node_modules 删了重新装一次依赖吧

**Comment by StarryFX** [comment]
嗯好的，我晚点试下，有问题再请教你，感谢回答

**Comment by StarryFX** [comment]
@Tsuk1ko 问题已找出，电脑之前蓝屏损坏了node（文件无法读取也无法删除），导致启动异常，正在修复中

**Comment by sakurayun** [comment]
啊这，我也是相同情况，但是依赖重装了多次还是无法正常启动

**Comment by StarryFX** [comment]
@sakurayun 文件残留？或者换个盘重新安装，回退node版本之类的，先试下看看
